reflect.js

```diff
@@ -658,6 +678,7 @@ Validator.prototype = {
       throw new TypeError("cannot report a non-configurable descriptor "+
                           "for configurable or non-existent property '"+name+"'");
     }
+    */
```

Comment:
Can you comment on why this invariant check must be removed? Also, since this change has nothing to do with this PR's original request (printing deprecation warning only once) it should probably go in its own PR?